### ISSUE TITLE
File hash check for local fs and https

### ISSUE THREAD

**Issue by madiganz** [seung-lab/cloud-files#76]
Not sure if this is the place for asking questions, but I see that for network robustness, when transferring a file, the hash is checked to make sure the destination matches the source for GCS and S3. Is there a reason why this check isn't also done for local file system and https transfers?

**Comment by madiganz** [comment]
Yes that answers my question and makes sense for local file systems. I am not sure if there is a standard that most web servers follow, so I think sounds like not checking for now is the way to go

**Comment by william-silversmith** [comment]
Yea, I'm also happy to build in some special handling for important systems, but I think they need to be identified to me first.
